i18n/languages/iyzico-woocommerce-checkout-form-tr_TR.po

```diff
@@ -0,0 +1,92 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.11\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: tr_TR\n"
+
+# This file is distributed under the GNU General Public License v2 or later.
+msgid "iyzico Checkout"
+msgstr "iyzico Ödeme"
+
+msgid "Proceed to iyzico checkout"
+msgstr "Ödeme iyzico geçin"
+
+msgid "Enable/Disable"
+msgstr "Enable / Disable"
+
+msgid "Enable iyzico checkout"
+msgstr "Iyzico ödeme etkinleştirin"
+
+msgid "Title"
+msgstr "başlık"
+
+msgid "This message will show to the user during checkout."
+msgstr "Bu mesaj ödeme sırasında kullanıcıya gösterecektir."
+
+msgid "Description"
+msgstr "tanım"
+
+msgid "Installment"
+msgstr "taksit"
+
+msgid "Payment Mode"
+msgstr "Ödeme Modu"
+
+msgid "Live"
+msgstr "Canlı"
+
+msgid "Live Merchant API ID"
+msgstr "Üye İşyeri Canlı API Anahtarı"
+
+msgid "Live Merchant Secret Key"
+msgstr "Üye İşyeri Canlı Güvenlik Anahtarı"
+
+msgid "Please use the iyzico panel for partial refund. Iyzico Panel link https://merchant.iyzipay.com/login"
+msgstr "Parçalı iade işlemi için lütfen iyzico panelini kullanın  Iyzico Panel link https://merchant.iyzipay.com/login"
+
+msgid "Thank you for your order, please click the button below to pay with iyzico Checkout."
+msgstr "Siparişiniz için teşekkür ederiz, iyzico Ödeme ile ödeme için aşağıdaki düğmeyi tıklayın."
+
+msgid "Thank you for shopping with us. Your account has been charged and your transaction is successful."
+msgstr "Bizimle alışveriş için teşekkür ederiz. Hesabınız suçlanıyor ve işlem başarılı."
+
+msgid "Installment Fee"
+msgstr "Taksit Ücreti"
+
+msgid "Iyzico Transaction ID"
+msgstr "Iyzico Transaction ID"
+
+msgid "Thank you for shopping with us. However, the transaction has been declined."
+msgstr "Bizimle alışveriş için teşekkür ederiz. Ancak, işlem reddedildi."
+
+msgid "Transaction ERROR"
+msgstr "İşlem HATA"
+
+msgid "Error"
+msgstr "hata"
+
+msgid "iyzico Checkout does not support your store currency."
+msgstr "iyzico ödeme formu para biriminizi desteklememektedir."
+
+msgid "Payment successful."
+msgstr "Ödeme başarılı"
+
+msgid "Invalid Token"
+msgstr "Geçersiz Token"
+
+msgid "Invalid request"
+msgstr "Geçersiz istek"
+
+msgid "Security error. Illegal access detected."
+msgstr "Güvenlik Hatası (illegal access detected.)"
+
+msgid "Please sure the currency value in your payment request is one of the (USD, EUR, GBP, IRR, TL) valid values."
```

Comment:
"Please make sure that the currency value in your payment request is one of the (USD, EUR, GBP, IRR, TL) valid values."